### DISCUSSION TITLE
如果可以保持原文件不动，重新生成按规则整理好的硬链接文件就好了

### DISCUSSION THREAD

**Discussion by fengzhiwen1029** [Yuukiy/JavSP#454] [想法 | Ideas]
很多时候资源是从bt pt站点下载的，为了做种不想破坏原有文件结构，但是那样资源又太混乱，所以会采用硬链接的方式整理资源。

希望可以加入可选功能，即：①不要移动原文件；②整理后的文件通过硬链接链接到原来的文件③识别目录下的srt字幕文件。

**Comment by fengzhiwen1029** [comment]
虽然整理好了我的存货让我很满足，但是也让我的五十多个保种文件都完蛋了。

**Comment by fengzhiwen1029** [comment]
硬链接（英语：hard link）是[计算机文件系统]中的多个文件平等地共享同一个文件存储单元。硬链接必须在同一个文件系统中；一般用户权限下的硬链接只能用于文件，不能用于目录，因为其父目录就有歧义了。删除一个文件名字后，还可以用其它名字继续访问该文件。硬链接只能用于同一个文件系统（对于NTFS是限制于同一个分区）。不能用于不存在的文件。

**Comment by glyh** [comment]
https://github.com/Yuukiy/JavSP/issues/434
